include/swift/AST/Types.h

```diff
@@ -2501,22 +2534,35 @@ END_CAN_TYPE_WRAPPER(AnyFunctionType, Type)
 ///
 /// For example:
 ///   let x : (Float, Int) -> Int
-class FunctionType : public AnyFunctionType {
+class alignas(AnyFunctionType::Param) FunctionType final :
```

Comment:
But... fixing the alignment of `FunctionType` doesn't help the trailing objects be properly aligned. You don't need this `alignas` change.